s3cmd

```diff
@@ -1920,6 +1923,15 @@ def cmd_sync_local2remote(args):
         error(u"or disable encryption with --no-encrypt parameter.")
         sys.exit(EX_USAGE)
 
+    # Disable md5 checks if using SSE-C. Add mtime check
+    if cfg.sse_customer_key or cfg.sse_copy_source_customer_key:
+        try:
+            cfg.sync_checks.remove("md5")
+        except Exception:
+            pass
+        if cfg.sync_checks.count("mtime") == 0:
```

Comment:
I think that you can simply do:
`if "mtime" not in cfg.sync_checks:`

Reply:
Yeah, wasn't changing that part from fork. Changed